src/pages/DashboardPage/DashboardPage.types.ts

```diff
@@ -0,0 +1,39 @@
+import type { LeaderboardUser } from '../../components/LeaderboardPage/LeaderboardPage.types';
```

Comment:
This file imports LeaderboardUser from the LeaderboardPage types. Importing types across feature boundaries can create accidental circular dependencies if not managed carefully.

**Sugerencia:**
```typescript
If types are shared across multiple areas, consider having a small shared-types module (e.g., src/types/) to avoid potential circular dependencies between components and pages.
```